scim2/tests/test.bal

```diff
@@ -1,10 +1,9 @@
 package tests;
 
 import ballerina/test;
-import scim2;
 import ballerina/io;
 
-endpoint scim2:Scim2Endpoint scimEP {
+endpoint  Scim2Endpoint scimEP {
     oauthClientConfig:{
                           accessToken:"60e49e81-8a70-340e-b017-ab6fc71ec2cb",
```

Comment:
Can we get this configuration details from ballerina.conf file or constant file?

Reply:
+1. Let's create a issue for this.